src/Strategy/Elixir/MixTree.hs

```diff
@@ -59,41 +71,37 @@ missingDepVersionsMsg = "Some of dependencies versions were not resolved from `m
 analyze' :: (Has Exec sig m, Has Diagnostics sig m) => Path Abs Dir -> m (Graphing Dependency)
 analyze' dir = do
   -- Get all dependencies
-  depsAllEnvTree <- context "Identifying relationship among dependencies" $ execParser mixTreeCmdOutputParser dir $ mixDepTreeCmd (Nothing)
-  depsAllResolved <- context "Inferring dependencies versioning" $ execParser mixDepsCmdOutputParser dir mixDepCmd
+  depsAllEnvTree <- context "Identifying relationship among dependencies" $ execParser mixTreeCmdOutputParser dir mixDepTreeCmd <&> supportedMixDeps
+  depsAllResolved <- context "Inferring dependencies versioning" $ execParser mixDepsCmdOutputParser dir mixDepCmd <&> Map.filter (supportedSCM <$> depResolvedSCM)
```

Comment:
I'd rather not see `<&>` used in the codebase, we don't really use it or `&` anywhere else.
Also, the `<$>` in `(supportedSCM <$> depResolvedSCM)` is just `.`.  I mentioned it on the poetry PR, but `-> a` has a functor instance, where `fmap = (.)`.  I've corrected that in my suggestion.

```suggestion
  depsAllEnvTree <- 
    context "Identifying relationship among dependencies" $
      supportedMixDeps <$> execParser mixTreeCmdOutputParser dir mixDepTreeCmd
  depsAllResolved <-
    context "Inferring dependencies versioning" $
      Map.filter (supportedSCM . depResolvedSCM)
        execParser mixDepsCmdOutputParser dir mixDepCmd
```

Reply:
+style guide https://github.com/fossas/team-analysis/issues/671